.gitignore

```diff
@@ -9,3 +9,5 @@ npm-debug.log
 .DS_Store
 .screenshots/
 
+*.iml
+*.idea
```

Comment:
This is IDE specific, so we should rather have these exceptions locally, on a global .gitignore file.

Reply:
On the other hand, we have it on other projects, so perhaps we can keep it, just rename `*.idea` to `.idea`, since it's a folder.